src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(x) {
+      calculator.result += x;
+    },
+    subtract(x) {
+      calculator.result -= x;
+    },
+    divide(x) {
+      calculator.result /= x;
+    },
+    multiply(x) {
+      calculator.result *= x;
+    },
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
+    },
+
+    operate(callback, x) {
+      callback(x);
+
+      return calculator;
+    },
+  };
+
+  return calculator;
```

Comment:
check task description: Each operate call takes a callback and a number and sets the appropriate value to the result property.
you should set needed value to the result in operate method but you  setting it in each callback that's not correct, try ti fix it